app/services/achiever/connection.rb

```diff
@@ -5,7 +5,7 @@ def self.api
     Faraday.new(url: ENV.fetch('ACHIEVER_V2_ENDPOINT')) do |conn|
       conn.adapter :net_http
       conn.request(:basic_auth, ENV.fetch('ACHIEVER_V2_USERNAME'), ENV.fetch('ACHIEVER_V2_PASSWORD'))
-      conn.proxy = ENV.fetch('PROXY_URL')
+      conn.proxy = ENV.fetch('PROXY_URL') # set PROXY_URL='' if you don't need a proxy
```

Comment:
unless I'm misunderstanding the intention, you could just add a fallback to an empty string `ENV.fetch('PROXY_URL', '')` that way it doesn't need to be defined